index.html

```diff
@@ -1114,51 +1279,60 @@ <h1>Verification</h1>
       <section>
         <h3>Syntax</h3>
         <ul>
-          <li>Document is syntactically valid (e.g. JSON, JSON-LD).</li>
+          <li>The document is syntactically valid (e.g. JSON, JSON-LD).</li>
         </ul>
       </section>
       <section>
         <h3>Credential</h3>
         <ul>
-          <li>Required properties are present. For example, for a
-          Credential, <code>type</code> and <code>claim</code> are
-          required.</li>
-          <li>Property values match expectations described in the
-          specification. For example, the document type for a verifiable
-          credential must contain the class "Credential".</li>
+          <li>Required properties MUST be present. For example, for a
+          <a>verifiable credential</a>, <code>type</code>
+          and <code>proof</code> properties are required.</li>
+          <li>Property values MUST match expectations described in this
+          specification. For example, the document <code>type</code> property
+          for a verifiable credential MUST contain the class
+          <code>VerifiableCredential</code>.</li>
         </ul>
       </section>
       <section>
         <h3>Issuer</h3>
         <ul>
-          <li>The <code>issuer</code> id must match expectations. Likely,
-          that means it is the id of a known and trusted <a>verifiable
-          presentation</a>.</li>
-          <li>Recent metadata about the <code>issuer</code> which was published
-          by the issuer MUST be available.</li>
+          <li>The value associated with the <code>issuer</code>
+          property MUST identify an <a>issuer</a> that is known to and trusted
+          by the <a>verifier</a>.</li>
+          <li>Pertinent metadata about the <code>issuer</code> MUST be
+          available to the <a>verifier</a>. For example, an issuer may publish
+          information that contains the public keys they use to digitally
+          sign <a>verifiable credentials</a> that they have issued. This
+          metadata is pertinent when checking the signatures on the
```

Comment:
"signatures" => "proofs"

Reply:
Fixed in 94904807a4f23cc7922d44de9f9c6719eddb2b75.